pppwn.py

```diff
@@ -838,6 +838,8 @@ def main():
         offs = OffsetsFirmware_900()
     elif args.fw == '1000':
         offs = OffsetsFirmware_1000()
+    elif args.fw == '1001':
```

Comment:
`parser.add_argument` should also be altered then, and also that check in the Makefiles (actually it should probably be dropped, it's gonna fail with wrong values anyways) and the `#if` in the headers

Reply:
@loskutov Ehhh sorry, didn't notice that. Been busy grieving because I misspelled the commit message :")).

As for the check in the make file, probably removing it is a good idea. We should include the alias in the header files as well.